drgn/helpers/linux/slab.py

```diff
@@ -258,25 +258,59 @@ def __init__(self, slab_cache: Object) -> None:
             freelist_random = slab_cache.random.value_()
         except AttributeError:
 
-            def _freelist_dereference(ptr_addr: int) -> int:
+            def _simple_dereference(ptr_addr: int) -> int:
                 return self._prog.read_word(ptr_addr)
 
+            self._freelist_dereference = _simple_dereference
+
         else:
             ulong_size = sizeof(self._prog.type("unsigned long"))
 
-            def _freelist_dereference(ptr_addr: int) -> int:
+            # Since Linux kernel commit 1ad53d9fa3f6 ("slub: improve bit
+            # diffusion for freelist ptr obfuscation"), a swab() was added to
```

Comment:
It's not as meaningful since it was backported to stable, but it's still useful context to know the original kernel version this was changed in:

```suggestion
            # Since Linux kernel commit 1ad53d9fa3f6 ("slub: improve bit
            # diffusion for freelist ptr obfuscation") (in v5.7), a swab() was added to
```